spec/acceptance/suites/default/00_default_spec.rb

```diff
@@ -16,6 +16,31 @@
       # (...but maybe we need it now?)
       install_simp_repos(host)
     end
+
+    it 'sets up dnsmasq' do
```

Comment:
Why is it necessary to set up a DNS cache now?

Reply:
It's what the ACA documentation says to do. No idea if it makes a difference but I wanted to match the docs.